tests/conftest.py

```diff
@@ -2517,7 +2518,8 @@ def finalizer():
 
     # Checks OCP version
     ocp_version = get_running_ocp_version()
-    logging_channel = "stable" if ocp_version >= "4.7" else ocp_version
+    ocp_version = Version.coerce(ocp_version)
```

Comment:
why not directly use `get_semantic_ocp_version_from_config`?

Reply:
done